### PR TITLE
docs: Jbang install option

### PR DESCRIPTION
**Pull request by quintesse**
Added explanation how to install Jbang using Jbang is the first option, thereby promoting the option that has no additional dependencies on package managers that are not available by default on the user's system.

### CODE REVIEW

File: readme.adoc [hunk 1]
```diff
@@ -94,9 +94,29 @@ Once you have installed from one of the below approaches it is recommended you r
 to have it setup your `PATH` to include jbang app scripts + it will on operating systems that supports
 it setup a `j!` alias you can use instead of `jbang`.
 
+=== Using Jbang
+
+Perhaps the coolest way to install Jbang is using Jbang itself.
```
Comment (maxandersen) [inline readme.adoc:99]:
```suggestion
The simplest way to install `jbang` is using Jbang itself.
```

Comment (quintesse) [reply readme.adoc]: Aww shucks, I liked "coolest" better :stuck_out_tongue:

Comment (maxandersen) [reply readme.adoc]: "Perhaps" doesn't exactly oozes confidence :)

Comment (quintesse) [reply readme.adoc]: Well "coolness" is of course a matter of taste :-)

**Pull request merged**